### DISCUSSION TITLE
Suppressing OSSEC under SOC Alerts

### DISCUSSION THREAD

**Discussion by jsdjjsd** [Security-Onion-Solutions/securityonion#4356] [Unsupported Versions]
How can I suppress the OSSEC-related alerts in Alerts? I only want my analysts to see network related alerts. Can this be done by tweaking the queries in soc.json or is there any other recommended way of doing this?

**Comment by weslambert** [comment]
Right now, you can filter them out from the Alerts view when you are viewing the alerts.  Otherwise, you would need to suppress each rule, override all rules in local_rules.xml, or adjust the log_alert_level in ossec.conf.

**Reply by mhgoff** [reply]
I have tried to suppress the rules, I have tried to change the log_alert_level but I guess I am doing something wrong.  What should the log_alert_level be set to in order to NOT show "low" alerts?
